packages/server/graphql/public/mutations/startRetrospective.ts

```diff
@@ -47,10 +47,18 @@ const startRetrospective: MutationResolvers['startRetrospective'] = async (
     videoMeetingURL
   } = meetingSettings as typeof meetingSettings & {meetingType: 'retrospective'}
   const selectedTemplateId = meetingSettings.selectedTemplateId || 'workingStuckTemplate'
+  const startTime=new Date(
+    rrule!.dtstart.year,
+    rrule!.dtstart.month - 1,
+    rrule!.dtstart.day,
+    rrule!.dtstart.hour,
+    rrule!.dtstart.minute,
+    rrule!.dtstart.second
+  );
```

Comment:
-1 With your current changes in the client, this should just be `rrule.after(new Date())`.
`startTeamPrompt` needs the same treatment.

Actually, if we start the meeting now, then it will be closed on `startTime` and a new meeting opened. So this is wrong and `new Date()` was correct.

Reply:
> Actually, if we start the meeting now, then it will be closed on startTime and a new meeting opened. So this is wrong and new Date() was correct.

But in the `createMeetingSeriesTitle` mutation if you pass the `new Date()` instead of `startTime` then you'll never have a recurrent meeting that begins with the date that user opted for right? It'll always be today.

Reply:
No, the next meeting in the series will be started at `startTime` and will get a new `createMeetingSeriesTitle` with the `startTime`.
There is no complete clarity how we can solve this issue, so it's not ideal to be picked up by you. We need to do some UI/UX work first.

Reply:
Sure then, I'll work on something else. Converting this to draft, will reopen with updated PR once we have some clarity.